build.sbt

```diff
@@ -308,20 +308,22 @@ lazy val databaseBenchmarks = (project in file("benchmarks/database"))
     name := "database",
     commonSettingsScala3,
     libraryDependencies ++= Seq(
-      "com.github.jnr" % "jnr-posix" % "3.1.18",
+      "com.github.jnr" % "jnr-posix" % "3.1.19",
       "org.apache.commons" % "commons-math3" % commonsMath3Version,
-      "org.agrona" % "agrona" % "1.19.2",
+      "org.agrona" % "agrona" % "1.21.1",
       // Database libraries.
-      "org.mapdb" % "mapdb" % "3.0.10",
+      "org.mapdb" % "mapdb" % "3.1.0",
       "com.h2database" % "h2-mvstore" % "2.2.224",
-      "net.openhft" % "chronicle-map" % "3.22.9",
+      "net.openhft" % "chronicle-map" % "3.23.5",
+      "net.openhft" % "affinity" % "3.23.3", // required by chronicle-map 3.23.5
```

Comment:
Doesn't `chronicle-map` pickup the `affinity` dependency automatically?

Reply:
For me it didn't build anymore, they have a strange dependency management... ymmv, just remove this line and try yourself 😅

Reply:
Various missing BOMs (or what not) was the reason I did not update to the latest versions previously (for release 0.15) and stuck to the versions SBT was able to build with.